lib/rules/disallow-multiple-var-decl.js

```diff
@@ -1,21 +1,23 @@
 /**
  * Disallows multiple `var` declaration (except for-loop).
  *
- * Types: `Boolean` or `String`
+ * Types: `Object`
```

Comment:
@stefanosala Yeah, just we should be more implicit. We have a lot of rules with true value with default behaviour (so I'm not sure we should deprecate it), but `string` values is deprecated and we can note it right now.

We have an issue about new format for 2.0 where we going to rework all rules (that's why we trying to prepare them to object form).